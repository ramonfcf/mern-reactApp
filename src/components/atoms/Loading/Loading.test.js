import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import Loading from "./Loading";

describe("Loading component", () => {
  it("matches snapshot", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
