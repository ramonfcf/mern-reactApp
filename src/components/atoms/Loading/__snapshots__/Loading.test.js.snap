// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component matches snapshot 1`] = `
<div>
  <div
    class="d-flex justify-content-center my-5"
  >
    <div
      class="spinner-border"
      role="status"
    >
      <span
        class="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
